main.py

```diff
@@ -10,14 +10,15 @@
 import logging
 import os
 import message
+import sys
 
 
 class Run(object):
     percentage_completed = -1
     last_log_line = ""
 
     def __init__(self):
-        self.torrent = torrent.Torrent().load_from_path("torrent.torrent")
+        self.torrent = torrent.Torrent().load_from_path(os.path.join('torrents', sys.argv[1]))
```

Comment:
This must be mentioned in the readme as well.

Reply:
This is not needed anymore, already done in master